core/domain/question_services_test.py

```diff
@@ -935,6 +935,7 @@ def test_untag_deleted_misconceptions_correctly_on_updating_skill(self):
             change_list, 'Delete misconceptions.')
         skill_fetchers.get_skill_by_id('skillid12345')
         self.process_and_flush_pending_tasks()
```

Comment:
is this still needed?

Reply:
Ah good catch this was added as part of a merge. Removed.